include/dex_mon_measures.h

```diff
@@ -12,39 +12,39 @@ struct ZknHeightWeight {
     u32 *unk_14;
 }; // size=0x18
 
-struct ZknHeightWeight *ZknData_Create(HeapID heapId);
-void ZknData_Delete(struct ZknHeightWeight *zkn);
-void ZknData_LoadAll(struct ZknHeightWeight *zkn, int mode, HeapID heapId);
-void ZknData_UnloadAll(struct ZknHeightWeight *zkn);
-u32 ZknData_GetHeight(struct ZknHeightWeight *zkn, int species);
-u32 ZknData_GetWeight(struct ZknHeightWeight *zkn, int species);
+struct PokedexData *PokedexData_Create(HeapID heapId);
+void PokedexData_Delete(struct PokedexData *zkn);
+void PokedexData_LoadAll(struct PokedexData *zkn, int mode, HeapID heapId);
+void PokedexData_UnloadAll(struct PokedexData *zkn);
+u32 PokedexData_GetHeight(struct PokedexData *zkn, int species);
+u32 PokedexData_GetWeight(struct PokedexData *zkn, int species);
 void SetDexBanksByGiratinaForme(int forme);
-NarcId GetDexZknDataNarcID(void);
+NarcId GetDexDataNarcID(void);
```

Comment:
I'd prefer this to be GetPokedexDataNarcID personally, matches the other ones that use Pokedex instead of Dex

Reply:
kk will change it